consensus/broadcast/metaChainMessenger_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/multiversx/mx-chain-go/common"
-	//"github.com/multiversx/mx-chain-go/consensus"
+	cons "github.com/multiversx/mx-chain-go/consensus"
```

Comment:
this should be simply without alias, thus using it as consensus further

as this will be in fonclict with import of testscommon/consensus, I would add an alias for that as consensusMocks or similar